api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirPatientServiceImpl.java

```diff
@@ -106,4 +107,26 @@ public IBundleProvider searchForPatients(StringAndListParam name, StringAndListP
 		
 		return searchQuery.getQueryResults(theParams, dao, translator, searchQueryInclude);
 	}
+	
+	@Override
+	@Transactional(readOnly = true)
+	public IBundleProvider getPatientEverything(TokenAndListParam patientId) {
+		SearchParameterMap theParams = new SearchParameterMap()
+		        .addParameter(FhirConstants.EVERYTHING_SEARCH_HANDLER, new StringParam())
```

Comment:
Since the value here doesn't matter and `SearchParameterMap` takes any `Serializable` object as a valid value, this is a more sensible implementation:

```suggestion
		        .addParameter(FhirConstants.EVERYTHING_SEARCH_HANDLER, "")
```

(The idea for requesting this change is that `new StringParam()` creates a object whereas `""` already exists in the string constant pool, so we save a tiny fraction of memory. It also, I think, makes it a little clearer that the value here isn't meaningful).

Reply:
@ibacher Should we create a PR to correct this for `$lastn` operations because I had done a similar thing there?

Reply:
Yeah, I suppose that's a good idea.